integration-tests/src/minio.rs

```diff
@@ -0,0 +1,117 @@
+        envs.insert("MINIO_ROOT_USER".into(), "admin".into());
+        envs.insert("MINIO_ROOT_PASSWORD".into(), "password".into());
+        envs.insert("MINIO_NOTIFY_KAFKA_ENABLE_BOMBASTIC".into(), "on".into());
+        envs.insert("MINIO_NOTIFY_KAFKA_BROKERS_BOMBASTIC".into(), "kafka:9094".into());
+        envs.insert("MINIO_NOTIFY_KAFKA_TOPIC_BOMBASTIC".into(), "sbom-stored".into());
+        envs.insert("MINIO_NOTIFY_KAFKA_ENABLE_VEXINATION".into(), "on".into());
+        envs.insert("MINIO_NOTIFY_KAFKA_BROKERS_VEXINATION".into(), "kafka:9094".into());
+        envs.insert("MINIO_NOTIFY_KAFKA_TOPIC_VEXINATION".into(), "vex-stored".into());
+        envs
+    };
+}
+
+#[derive(Clone, Debug)]
+pub struct MinIOArgs {
+    args: Vec<String>,
+}
+
+impl Default for MinIOArgs {
+    fn default() -> Self {
+        Self {
+            args: vec![
+                "server".into(),
+                "--console-address".into(),
+                ":9001".into(),
+                "/data1".into(),
+            ],
+        }
+    }
+}
+
+impl ImageArgs for MinIOArgs {
+    fn into_iterator(self) -> Box<dyn Iterator<Item = String>> {
+        Box::new(self.args.into_iter())
+    }
+}
+
+impl Image for MinIO {
+    type Args = MinIOArgs;
+
+    fn name(&self) -> String {
+        String::from("quay.io/minio/minio")
+    }
+
+    fn tag(&self) -> String {
+        String::from("RELEASE.2023-06-19T19-52-50Z")
+    }
+
+    fn ready_conditions(&self) -> Vec<testcontainers::core::WaitFor> {
+        vec![WaitFor::message_on_stdout("API:")]
+    }
+
+    fn env_vars(&self) -> Box<dyn Iterator<Item = (&String, &String)> + '_> {
+        Box::new(ENV.iter())
+    }
+
+    fn expose_ports(&self) -> Vec<u16> {
+        vec![9000, 9001]
+    }
+}
+
+pub struct MinIOCli;
+
+impl Image for MinIOCli {
+    type Args = MinIOCliArgs;
+
+    fn entrypoint(&self) -> Option<String> {
+        Some("/bin/sh".to_string())
+    }
+
+    fn name(&self) -> String {
+        String::from("quay.io/minio/mc")
+    }
+
+    fn tag(&self) -> String {
+        String::from("RELEASE.2023-06-19T19-31-19Z")
+    }
+
+    fn ready_conditions(&self) -> Vec<testcontainers::core::WaitFor> {
+        Vec::new()
+    }
+}
+
+#[derive(Clone, Debug)]
+pub struct MinIOCliArgs {
+    args: Vec<String>,
+}
+
+impl Default for MinIOCliArgs {
+    fn default() -> Self {
+        const COMMANDS: &str = r###"/usr/bin/mc config host add myminio http://minio:9000 admin password;
+/usr/bin/mc mb myminio/bombastic || true;
+/usr/bin/mc policy download myminio/bombastic;
+/usr/bin/mc event add myminio/bombastic arn:minio:sqs::BOMBASTIC:kafka --event "put,delete";
+/usr/bin/mc mb myminio/vexination || true;
+/usr/bin/mc policy download myminio/vexination;
+/usr/bin/mc event add myminio/vexination arn:minio:sqs::VEXINATION:kafka --event put;
+/usr/bin/mc admin service restart myminio;"###;
+        Self {
+            args: vec!["-c".into(), COMMANDS.to_string()],
+        }
+    }
+}
```

Comment:
Again, this seems counter to relying on the compose.yaml file, which also has these commands.